sharktank/tests/models/t5/t5_test.py

```diff
@@ -480,19 +480,16 @@ def runTestV1_1CompareIreeAgainstTorchEager(
             assert_close=assert_t5_encoder_state_close,
         )
 
-    @skip(
-        reason=(
-            "The test hangs. Probably during compilation or IREE module "
-            "execution. We can't determine easily what is going on as running "
-            "tests in parallel with pyest-xdist is incompatible with capture "
-            "disabling with --capture=no. No live logs are available from the CI."
-            " TODO: investigate"
-        )
-    )
     @parameterized.expand(
         [
             (torch.float32, torch.float64, 1e-5, 1e-5),
-            (torch.bfloat16, torch.float64, 2e-2, 1e-2),
+            (
+                torch.bfloat16,
+                torch.float64,
+                2e-2,
+                1e-2,
+                "ISSUE: https://github.com/nod-ai/shark-ai/issues/1689",
+            ),  # mark.xfail
```

Comment:
You should use this pattern for xfailing for a specific parameter set
https://stackoverflow.com/a/30575822/324204

Reply:
Also use our xfail with a regex match.
https://github.com/nod-ai/shark-ai/blob/26e83175729864d159cbfcd56f17e322ead28d51/sharktank/sharktank/utils/testing.py#L656

Reply:
> missed that there is a param wrapper in parametrized lib too.
https://github.com/wolever/parameterized/blob/b9f6a640452bcfdea08efc4badfe5bfad043f099/parameterized/parameterized.py#L161

seems like this is also just a wrapper for args kwargs unlike pytest.param which can mark tests too.

Reply:
The match makes sure that the exception message does not change unexpectedly. If the test starts failing due to another reason. This is needed because some tests were failing during compilation, but then before they got fixed they started failing with another compiler bug.
Like that we even don't check if the exception type is as expected.

Reply:
I changed the atol as it is too stringent.

Reply:
Regarding the problems with xfailing a parametrized test here is an issue https://github.com/nod-ai/shark-ai/issues/1708.